README.md

```diff
@@ -133,12 +143,207 @@ Want to help out? Pull requests (PRs) are always welcome! If you want to help ou
+---
+```
+
+![](./static/images/front-matter-variables-title.png)
+
+#### Description
+
+The `description` variable defines what is in the [meta `<description>`](https://moz.com/learn/seo/meta-description) tag within the `<head>` tag of this page's HTML.  This description often shows up in search engine results, and social network embeds. This description is meant to give the reader an idea of the content on this page and how it relates to their search query.
+
+```YAML
+---
+description: "The Filecoin Network is made with storage providers and clients. They make deals and contribute to [...]"
+---
+```
+
+![](./static/images/front-matter-variables-description.png)
+
+#### Lead
+
+The `lead` variable defines the content of the first paragraph on a page. This is usually an introduction, informing the reader what this page is referring to, what they're about to learn, and any prerequisites for understanding the content on this page. Often, the content of this variable is the same as the `description` variable.
+
+```YAML
+---
+lead: "The Filecoin Network is made with storage providers and clients. They make deals and contribute to [...]"
+---
+```
+
+![](./static/images/front-matter-variables-lead.png)
+
+#### Weight
+
+The `weight` variable defines where this page or menu item should be in a menu. The lower the number, the closer to the start of the menu this page will be. If set, `weight` should be non-zero, as `0` is interpreted as an unset weight. There is no upper limit for a weight value.
+
+In the top-bar menu, a lower number will cause the menu item to be further to the left in a regular view or further to the top in a mobile view.
+
+This example is from the `/config/_default/menus/menus.en.toml` file:
+
+```
+[[main]]
+  name = "About Filecoin"
+  url = "/about-filecoin/what-is-filecoin"
+  weight = 10
+  
+[[main]]
+  name = "Networks"
+  url = "/networks/overview"
+  weight = 20
+
+[[main]]
+  name = "Get started"
+  url = "/get-started/overview"
+  weight = 30
+```
+
+![](./static/images/front-matter-variables-weight-1.png)
+
+In the sidebar menu, a lower number will cause the menu item or page to be higher up.
+
+![](./static/images/front-matter-variables-weight-2.png)
+
+The weight of a page also defines the _next_ and _previous_ buttons at the bottom of the page. The _previous_ page will be the page with the closest weight _below_ the current page's weight. The _next_ page will be the page with the closest weight _above_ the current page's weight.
+
+![](./static/images/front-matter-variables-weight-3.png)
+
+#### Menu
+
+The `menu` variable defines which sidebar menu this page is assigned to, along with which sub-menu this page falls under. This variable is made of three parts:
+
+1. The `menu` delimiter. This tells Hugo that were are about to define the menu object for this page.
+1. The section/top-bar menu that this page falls under.
+1. The sub-menu within the sidebar that this page falls under.
+
+```YAML

```

Comment:
maybe we should add an image here corresponding to the code example so it is easier to follow visually?